h/search/config.py

```diff
@@ -11,13 +11,29 @@
 from __future__ import unicode_literals
 
 import binascii
+import elasticsearch1
+import elasticsearch
 import logging
 import os
 
-from elasticsearch1.exceptions import NotFoundError, RequestError
 
 log = logging.getLogger(__name__)
 
+ELASTICSEARCH1_VERSION = ".".join(str(item) for item in elasticsearch1.__version__)
+ELASTICSEARCH_VERSION = ".".join(str(item) for item in elasticsearch.__version__)
```

Comment:
This needs updating/simplifying to reflect the type change of `client.__version__`. It might not be necessary to declare these constants - tests could just reference `elasticsearch1.__version__` directly.